subprojects/bounded_context/user/infra/src/main/kotlin/club/staircrusher/user/infra/service/JWTTokenManager.kt

```diff
@@ -4,13 +4,15 @@ import club.staircrusher.user.domain.service.TokenManager
 import com.auth0.jwt.JWT
 import com.auth0.jwt.algorithms.Algorithm
 import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.springframework.stereotype.Component
 import java.time.Clock
 import java.time.Duration
 import java.util.Date
 import kotlin.reflect.KClass
 
+@Component
 class JWTTokenManager(
-    secret: String,
+    secret: String = "secret",
```

Comment:
```suggestion
    secret: String,
```

no way 

ㅋㅋ 뭔가 실수하기 좋아보여요

Reply:
일단 동작하게 하느라...